### ISSUE TITLE
Configurable contact colors

### ISSUE THREAD

**Issue by fredludlow** [arose/ngl#435]
The colors for different contact types are currently hardcoded in contacts.ts. I'd quite like to be able to change them -  would it be better to have a parameter for each, or some variation on Colormaker? (Or can I already do it with Colormaker somehow?)

**Comment by arose** [comment]
yeah, should be a colormaker (colormaker.contactColor), same for the radius
